src/vscode-split-layout/vscode-split-layout.ts

```diff
@@ -292,4 +331,8 @@ declare global {
   interface HTMLElementTagNameMap {
     'vscode-split-layout': VscodeSplitLayout;
   }
+
+  interface GlobalEventHandlersEventMap {
+    'vsc-split-layout-position-changed': VscSplitLayoutPositionChangedEvent;
```

Comment:
I would use the present tense for the event type. Native events use the present tense, which is also the convention in this codebase.